docker-compose.yml

```diff
@@ -454,3 +456,35 @@ services:
       - SSL_CERTFILE=/tls.cert
       - SSL_KEYFILE=/tls.key
     command: ["Fulcrum", "/fulcrum.conf"]
+
+  svix:
+    image: svix/svix-server
+    platform: linux/amd64
+    environment:
+      WAIT_FOR: "true" # We want to wait for the default services
+      SVIX_REDIS_DSN: "redis://svix-redis:6379"
+      SVIX_DB_DSN: "postgresql://postgres:postgres@svix-pg/postgres"
+      SVIX_JWT_SECRET: "8KjzRXrKkd9YFcNyqLSIY8JwiaCeRc6WK4UkMnSW"
+    ports:
+      - "8071:8071"
+    depends_on:
+      - svix-pg
+      - svix-redis
+
+  svix-pg:
+    image: postgres:13.4
+    volumes:
+      - "svix-pg-data:/var/lib/postgresql/data/"
+    environment:
+      POSTGRES_PASSWORD: postgres
+
+  svix-redis:
+    image: "redis:7-alpine"
+    # Enable persistence
+    command: "--save 60 500 --appendonly yes --appendfsync everysec"
+    volumes:
+      - "svix-redis-data:/data"
+
+volumes:
```

Comment:
This key will probably need to be added to the Quickstart `docker-compose.tmpl.yml`